SECURITY.md

```diff
@@ -0,0 +1,35 @@
+# Security
+
+Target takes the security of our software products and services seriously, which includes all source code repositories managed through [our GitHub organizations](https://opensource.target.com).
+
+If you believe you have found a security vulnerability in any Target-owned repository that meets Target's definition of a security vulnerability, please report it to us as described below.
+
+## Reporting Security Issues
+
+**Please do not report security vulnerabilities through public GitHub issues.**
+
+Instead, please report them to the Target Cyber Security team at https://security.target.com/vdp.
+
+If you prefer to submit without logging in, send an email to [security@target.com](mailto:security@target.com).
```

Comment:
VDP doesn't require a login

Reply:
super strange - i could have sworn there was something there before and mention of a bug bounty / security collaboration website.

@jaylindquist I'll adust